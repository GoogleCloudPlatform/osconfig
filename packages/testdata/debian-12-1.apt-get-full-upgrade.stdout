Inst tzdata [2025a-0+deb12u1] (2025b-0+deb12u1 Debian:12-updates/stable-updates [all])
Inst libgirepository-1.0-1 (1.74.0-3 Debian:12.10/stable [amd64])
Inst gir1.2-glib-2.0 (1.74.0-3 Debian:12.10/stable [amd64])
Inst google-cloud-cli [455.0.0-0] (520.0.0-0 cloud-sdk-bullseye:cloud-sdk-bookworm [amd64])
Inst google-cloud-cli-anthoscli (520.0.0-0 cloud-sdk-bullseye:cloud-sdk-bookworm [amd64])
Inst google-cloud-packages-archive-keyring [1.2-582007467] (1.2-629101324 google-compute-engine-bookworm-stable:google-compute-engine-bookworm-stable [all])
Inst google-guest-agent [1:20231115.00-g1] (1:20250327.01-g1 google-compute-engine-bookworm-stable:google-compute-engine-bookworm-stable [amd64])
Inst google-compute-engine-oslogin [1:20231004.00-g1+deb12] (1:20240701.00-g1+deb12 google-compute-engine-bookworm-stable:google-compute-engine-bookworm-stable [amd64])
Inst python3-gi (3.42.2-3+b1 Debian:12.10/stable [amd64])
Inst networkd-dispatcher (2.2.3-1 Debian:12.10/stable [all])
Inst libonig5 (6.9.8-1 Debian:12.10/stable [amd64])
Inst libjq1 (1.6-2.1 Debian:12.10/stable [amd64])
Inst jq (1.6-2.1 Debian:12.10/stable [amd64])
Inst google-compute-engine [1:20230801.00-g1] (1:20250207.00-g1 google-compute-engine-bookworm-stable:google-compute-engine-bookworm-stable [all])
Inst google-osconfig-agent [1:20231010.00-g1] (1:20250320.00-g1 google-compute-engine-bookworm-stable:google-compute-engine-bookworm-stable [amd64])
Conf tzdata (2025b-0+deb12u1 Debian:12-updates/stable-updates [all])
Conf libgirepository-1.0-1 (1.74.0-3 Debian:12.10/stable [amd64])
Conf gir1.2-glib-2.0 (1.74.0-3 Debian:12.10/stable [amd64])
Conf google-cloud-cli (520.0.0-0 cloud-sdk-bullseye:cloud-sdk-bookworm [amd64])
Conf google-cloud-cli-anthoscli (520.0.0-0 cloud-sdk-bullseye:cloud-sdk-bookworm [amd64])
Conf google-cloud-packages-archive-keyring (1.2-629101324 google-compute-engine-bookworm-stable:google-compute-engine-bookworm-stable [all])
Conf google-guest-agent (1:20250327.01-g1 google-compute-engine-bookworm-stable:google-compute-engine-bookworm-stable [amd64])
Conf google-compute-engine-oslogin (1:20240701.00-g1+deb12 google-compute-engine-bookworm-stable:google-compute-engine-bookworm-stable [amd64])
Conf python3-gi (3.42.2-3+b1 Debian:12.10/stable [amd64])
Conf networkd-dispatcher (2.2.3-1 Debian:12.10/stable [all])
Conf libonig5 (6.9.8-1 Debian:12.10/stable [amd64])
Conf libjq1 (1.6-2.1 Debian:12.10/stable [amd64])
Conf jq (1.6-2.1 Debian:12.10/stable [amd64])
Conf google-compute-engine (1:20250207.00-g1 google-compute-engine-bookworm-stable:google-compute-engine-bookworm-stable [all])
Conf google-osconfig-agent (1:20250320.00-g1 google-compute-engine-bookworm-stable:google-compute-engine-bookworm-stable [amd64])
