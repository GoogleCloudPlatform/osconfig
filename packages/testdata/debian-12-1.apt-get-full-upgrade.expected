[]*packages.PkgInfo{
    &packages.PkgInfo{
        Name:    "tzdata",
        Arch:    "all",
        RawArch: "",
        Version: "2025b-0+deb12u1",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "google-cloud-cli",
        Arch:    "x86_64",
        RawArch: "",
        Version: "520.0.0-0",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "google-cloud-packages-archive-keyring",
        Arch:    "all",
        RawArch: "",
        Version: "1.2-629101324",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "google-guest-agent",
        Arch:    "x86_64",
        RawArch: "",
        Version: "1:20250327.01-g1",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "google-compute-engine-oslogin",
        Arch:    "x86_64",
        RawArch: "",
        Version: "1:20240701.00-g1+deb12",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "google-compute-engine",
        Arch:    "all",
        RawArch: "",
        Version: "1:20250207.00-g1",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "google-osconfig-agent",
        Arch:    "x86_64",
        RawArch: "",
        Version: "1:20250320.00-g1",
        Source:  packages.Source{},
    },
}